package.json

```diff
@@ -11,7 +11,7 @@
     "cut"
   ],
   "dependencies": {
-    "good-listener": "^1.2.0",
+    "good-listener": "tobilen/good-listener#shadydom-polyfill",
```

Comment:
If you update the branch, every new install on clipboard.js will get your updates.
It is safer to lock this to a specific commit hash, or even better to publish your fork to npm and use that.